tensorboard/backend/application.py

```diff
@@ -265,7 +266,32 @@ def __init__(self, plugins, path_prefix=''):
         else:
           path = (self._path_prefix + DATA_PREFIX + PLUGIN_PREFIX + '/' +
                   plugin.plugin_name + route)
-        self.data_applications[path] = app
+
+        if path.endswith('/*'):
+          # Note we remove the '*' but leave the slash in place.
+          path = path[:-1]
+          if '*' in path:
+            # note we re-add the removed * in the format string
+            raise ValueError('Plugin %r handles invalid route %r*:  Only '
+                             'trailing wildcards are supported '
```

Comment:
Nit, and I’m not requesting a change or blocking you on this, but please
note that some Google style guides explicitly discourage this kind of
horizontal alignment, because it means that changing the first line
forces you to reindent all the other lines for consistency. e.g.:
<https://google.github.io/styleguide/jsguide.html#formatting-horizontal-alignment>

Over time, as large-scale changes happen, necessarily either the blame
becomes polluted or the indentation rots; e.g.:

https://github.com/tensorflow/tensorboard/blob/757628a814410574718623b0a881ae10855754c8/tensorboard/plugins/pr_curve/pr_curve_demo.py#L45-L46

That file changed from `tf.flags` to `tf.compat.v1.flags` to `flags`,
and the alignment was only correct in the original form.

(If we adopt Black, per #2466, then this will be enforced. :-) )

Reply:
Fair, but I just copied this from your https://github.com/tensorflow/tensorboard/blob/master/tensorboard/backend/application.py#L260 ;)

Anyway +100 to autoformatting.  I appreciate your rigor, but let's let automation handle this stuff :)  Many thanks for already doing the Prettier side of that!